netlify.toml

```diff
@@ -0,0 +1,4 @@
+[[headers]]
+  for = "/*"
+  [headers.values]
+    X-Frame-Options = "ALLOW"
```

Comment:
sorry, there is no allow directive here, should be allow-from, I'm not toml expert though 
https://www.keycdn.com/blog/x-frame-options#3-allow-from-uri-directive
maybe this will work:
`X-Frame-Options: ALLOW-FROM https://www.youtube.com/`

Reply:
trying this, will definitely want to squash and merge 😆